pkg/webhook/stats_reporter.go

```diff
@@ -65,6 +67,19 @@ func (r *reporter) ReportAdmissionRequest(response admissionReqRes, d time.Durat
 	return r.report(ctx, responseTimeInSecM.M(d.Seconds()))
 }
 
+// Captures req count metric, recording the count and the duration
+func (r *reporter) ReportMutationRequest(response mutationResponse, d time.Duration) error {
```

Comment:
Add to pkg/webhook/stats_reporter_test.go?

Reply:
Added TestMutationReportRequest to pkg/webhook/stats_reporter_test.go

Reply:
sorry i am confused, should these tests be part of #976?

Reply:
Since #976 is on top of this PR, once this PR merges, the tests can be updated accordingly.